odoo/addons/base/views/res_partner_views.xml

```diff
@@ -108,15 +108,21 @@
                     Potential duplicates:
                     <span invisible="not same_vat_partner_id or same_company_registry_partner_id">
                         <field name="same_vat_partner_id" context="{'show_address': False, 'show_vat': False}"/>
-                        (Same Tax ID <field name="vat_label"/>)
+                        (Same Tax ID <field name="vat_value" nolabel="1"/>)
                     </span>
                     <span invisible="same_vat_partner_id or not same_company_registry_partner_id">
                         <field name="same_company_registry_partner_id" context="{'show_address': False, 'show_vat': False}"/>
-                        (Same Company Registry <field name="company_registry_label"/>)
+                        (Same Company Registry <field name="company_registry_value" nolabel="1"/>)
```

Comment:
Same here but we'll have to add a context key `show_company_registry` and handle it [here](https://github.com/odoo/odoo/blob/8243c608d99247c8d259aa4b43f86b86b514f207/odoo/addons/base/models/res_partner.py#L905-L923).